nextflow.config

```diff
@@ -85,6 +85,13 @@ params {
     starindex_ref                 = "${params.genomes_base}/star"
     fusionreport_ref              = "${params.genomes_base}/fusion_report_db"
     ctatsplicing_cancer_introns   = "https://data.broadinstitute.org/Trinity/CTAT_RESOURCE_LIB/CANCER_SPLICING_LIB_SUPPLEMENT/cancer_introns.GRCh38.Jun232020.tsv.gz"
+    dfam_hmm                      = null
```

Comment:
For consistency: I would put all the defaults in `nextflow.config` rather than `nextflow_schema.json` since it is the primary source

Reply:
The defaults should be in both (otherwise linting will complain). The schema defaults will be mainly used for documentation and the config defaults will be used as the real defaults